pandas/tests/frame/test_constructors.py

```diff
@@ -717,6 +717,16 @@ def test_constructor_period_dict(self):
         assert df["a"].dtype == a.dtype
         assert df["b"].dtype == b.dtype
 
+        # list of periods
```

Comment:
make this a new test

Reply:
Can do!

Reply:
Done! Based on my previous PR, I'm guessing you wanted the new test parametrized, so I went ahead and did that too 😄